### PR TITLE
DE/ES/IT/PL/PT-EFS-control_panel_guide_updates

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.es-es.md [hunk 1]
```diff
@@ -41,8 +37,10 @@ Puede realizar varias acciones haciendo clic en el botón `..`{.action} en cada
 
 - **Modificar el volumen**: abre el apartado "[Información general](#modify_volume)" del volumen.
 - **Crear un snapshot**: abre la sección "[Copias de seguridad](#snapshots)" para realizar un snapshot manual del volumen.
+- **Restaurar el último snapshot** : Abre la sección «[Snapshots](#snapshots)» y una ventana para restaurar el volumen.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.es-es.md]:
```suggestion
- **Restaurar el último snapshot** : Abre la sección "[Snapshots](#snapshots)" y una ventana para restaurar el volumen.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.es-es.md [hunk 2]
```diff
@@ -150,4 +214,6 @@ Si no está familiarizado con la solución Enterprise File Storage, puede seguir
 
 Si necesita formación o asistencia técnica para implantar nuestras soluciones, póngase en contacto con su representante de ventas o haga clic en [este enlace](https://www.ovhcloud.com/es-es/professional-services/) para obtener un presupuesto y solicitar un análisis personalizado de su proyecto a nuestros expertos del equipo de Servicios Profesionales.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.es-es.md]:
```suggestion
Si necesita formación o asistencia técnica para implantar nuestras soluciones, póngase en contacto con su representante de ventas o haga clic en [este enlace](/links/professional-services) para obtener un presupuesto y solicitar un análisis personalizado de su proyecto a nuestros expertos del equipo de Servicios Profesionales.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.de-de.md [hunk 3]
```diff
@@ -150,4 +214,6 @@ Wenn Sie mit der Enterprise File Storage Lösung nicht noch nicht vertraut sind,
 
 Wenn Sie Schulungen oder technische Unterstützung bei der Implementierung unserer Lösungen benötigen, wenden Sie sich an Ihren Vertriebsmitarbeiter oder klicken Sie auf [diesen Link](https://www.ovhcloud.com/de/professional-services/), um einen Kostenvoranschlag zu erhalten und eine persönliche Analyse Ihres Projekts durch unsere Experten des Professional Services Teams anzufordern.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.de-de.md]:
```suggestion
Wenn Sie Schulungen oder technische Unterstützung bei der Implementierung unserer Lösungen benötigen, wenden Sie sich an Ihren Vertriebsmitarbeiter oder klicken Sie auf [diesen Link](/links/professional-services), um einen Kostenvoranschlag zu erhalten und eine persönliche Analyse Ihres Projekts durch unsere Experten des Professional Services Teams anzufordern.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.it-it.md [hunk 4]
```diff
@@ -150,4 +214,6 @@ Se non conosci la soluzione Enterprise File Storage, segui questi passaggi:
 
 Se avete bisogno di formazione o di assistenza tecnica per implementare le nostre soluzioni, contattate il vostro rappresentante o cliccate su [questo link](https://www.ovhcloud.com/it/professional-services/) per ottenere un preventivo e richiedere un'analisi personalizzata del vostro progetto da parte dei nostri esperti del team Professional Services.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.it-it.md]:
```suggestion
Se avete bisogno di formazione o di assistenza tecnica per implementare le nostre soluzioni, contattate il vostro rappresentante o cliccate su [questo link](/links/professional-services) per ottenere un preventivo e richiedere un'analisi personalizzata del vostro progetto da parte dei nostri esperti del team Professional Services.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 5]
```diff
@@ -150,4 +214,6 @@ Jeśli nie jesteś zaznajomiony z korzystaniem z rozwiązania Enterprise File St
 
 Jeśli potrzebujesz szkolenia lub pomocy technicznej w celu wdrożenia naszych rozwiązań, skontaktuj się z przedstawicielem handlowym lub kliknij [ten link](https://www.ovhcloud.com/pl/professional-services/), aby uzyskać wycenę i poprosić o spersonalizowaną analizę projektu od naszych ekspertów z zespołu Professional Services.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
Jeśli potrzebujesz szkolenia lub pomocy technicznej w celu wdrożenia naszych rozwiązań, skontaktuj się z przedstawicielem handlowym lub kliknij [ten link](/links/professional-services), aby uzyskać wycenę i poprosić o spersonalizowaną analizę projektu od naszych ekspertów z zespołu Professional Services.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pt-pt.md [hunk 6]
```diff
@@ -150,4 +214,6 @@ Se não está familiarizado com a utilização da solução Enterprise File Stor
 
 Se precisar de formação ou de assistência técnica para implementar as nossas soluções, contacte o seu representante comercial ou clique em [esta ligação](https://www.ovhcloud.com/pt/professional-services/) para obter um orçamento e solicitar uma análise personalizada do seu projecto aos nossos especialistas da equipa de Serviços Profissionais.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pt-pt.md:215]:
```suggestion
Se precisar de formação ou de assistência técnica para implementar as nossas soluções, contacte o seu representante comercial ou clique em [esta ligação](/links/professional-services) para obter um orçamento e solicitar uma análise personalizada do seu projecto aos nossos especialistas da equipa de Serviços Profissionais.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pt-pt.md [hunk 7]
```diff
@@ -41,8 +37,10 @@ Pode efetuar várias ações clicando no botão `...`{.action} em cada linha da
 
 - **Modificar o volume**: Abre a secção "[Informações gerais](#modify_volume)" do volume.
 - **Criar uma snapshot**: abra a secção "[Backups](#snapshots)" para efetuar uma snapshot manual do volume.
+- **Restaurar a última snapshot** : abre a secção « [Snapshots](#snapshots) » bem como uma janela que permite restaurar o volume.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pt-pt.md]:
```suggestion
- **Restaurar a última snapshot** : abre a secção "[Snapshots](#snapshots)" bem como uma janela que permite restaurar o volume.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 8]
```diff
@@ -51,80 +49,146 @@ Kliknij przycisk `Utwórz wolumen`{.action}. W nowym oknie, które się pojawi,
...
+Wypełnij pola, aby określić kryteria okresowości tworzenia snapshota. Musisz również podać prefiks dla snapshotów niezbędny do nadania im nazwy.
 
-Zaznacz nową regułę, aby ją dodać. Po dodaniu wszystkich reguł kliknij `Utwórz nowy Snapshot policy`{.action}.
+Szczegółowe informacje na temat każdej wartości można znaleźć, klikając ikonę znaku zapytania (`?`{.action}). Rozwijając sekcję `Przykład`{.action}, możesz wyświetlić dwa zestawy reguł wraz z wyjaśnieniem ich wyniku.
 
-[Wybierz wolumen](#manage_volume) i przejdź do zakładki `Snapshoty`{.action}, aby [stosować reguły](#snapshots).
+Zaznacz nową regułę, aby ją dodać. Po dodaniu wszystkich reguł kliknij przycisk `Utwórz nową politykę wykonywania snapshotów`{.action}.
 
-Aby usunąć politykę, w tabeli kliknij odpowiednią ikonę kosza.
+[Wybierz wolumen] (#manage_volume) i przejdź do zakładki `Snapshots`{.action}, aby [zastosować reguły] (#snapshots).
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
[Wybierz wolumen](#manage_volume) i przejdź do zakładki `Snapshots`{.action}, aby [zastosować reguły](#snapshots).
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 9]
```diff
@@ -51,80 +49,146 @@ Kliknij przycisk `Utwórz wolumen`{.action}. W nowym oknie, które się pojawi,
...
+Pojawi się nowe okno, w którym możesz wpisać nazwę i opis.
+
+Kliknij przycisk `Utwórz snapshot`{.action}, aby rozpocząć tworzenie.
+
+![Snapshot create window](images/manage_enterprise09.png){.thumbnail}
+
+### Zmiana snapshota <a name="modify_snapshot"></a>
 
-Aby ręcznie dodać nowy snapshot wolumenu do aktualnego stanu, kliknij przycisk `Operacje`{.action}, a następnie `Utwórz snapshot`{.action}.
+Możesz zmienić nazwę lub opis snapshota klikając na przycisk ``...`{.action} w tabeli, następnie kliknij na `Edytuj`{.action}. Otworzy się wówczas okno, w którym możesz wybrać nową nazwę i/lub opis.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
Możesz zmienić nazwę lub opis snapshota klikając na przycisk `...`{.action} w tabeli, następnie kliknij na `Edytuj`{.action}. Otworzy się wówczas okno, w którym możesz wybrać nową nazwę i/lub opis.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 10]
```diff
@@ -51,80 +49,146 @@ Kliknij przycisk `Utwórz wolumen`{.action}. W nowym oknie, które się pojawi,
...
-![Snapshoty](images/manage_enterprise05.png){.thumbnail}
+![Snapshots](images/manage_enterprise06.png){.thumbnail}
+
+W tej samej karcie można również wyświetlić wszystkie [reguły snapshotów] (#snapshot_policy) utworzone dla usługi i zastosować je do tego wolumenu.
+
+![Volume Snapshot Policy Management](images/manage_enterprise07.png){.thumbnail}
+
+Kliknij linię odpowiedniej reguły, aby wyświetlić szczegóły harmonogramu snapshotów. Wybierz politykę, klikając dedykowany przycisk wyboru, następnie kliknij przycisk `Zastosuj politykę`{.action} znajdujący się pod tabelą.
+
+Aby skonfigurować [zasady wykonywania snapshotów] (#snapshot_policy), wróć do sekcji [Zarządzanie wolumenami] (#instrukcje) i otwórz zakładkę `Snapshot policies`{.action}.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
Aby skonfigurować [zasady wykonywania snapshotów](#snapshot_policy), wróć do sekcji [Zarządzanie wolumenami](#instrukcje) i otwórz zakładkę `Snapshot policies`{.action}.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 11]
```diff
@@ -51,80 +49,146 @@ Kliknij przycisk `Utwórz wolumen`{.action}. W nowym oknie, które się pojawi,
...
+>
+> Snapshoty wykorzystują przestrzeń dyskową rozwiązania Enterprise File Storage. Domyślnie, 5% rozmiaru wolumenu jest zawsze zarezerwowane na snapshoty.
+>
+
 W zakładce `Snapshoty`{.action} wyszczególnione są wszystkie snapshoty utworzone dla wybranego wolumenu.
 
-![Snapshoty](images/manage_enterprise05.png){.thumbnail}
+![Snapshots](images/manage_enterprise06.png){.thumbnail}
+
+W tej samej karcie można również wyświetlić wszystkie [reguły snapshotów] (#snapshot_policy) utworzone dla usługi i zastosować je do tego wolumenu.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
W tej samej karcie można również wyświetlić wszystkie [reguły snapshotów](#snapshot_policy) utworzone dla usługi i zastosować je do tego wolumenu.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 12]
```diff
@@ -51,80 +49,146 @@ Kliknij przycisk `Utwórz wolumen`{.action}. W nowym oknie, które się pojawi,
...
+W zakładce `Informacje ogólne`{.action} wyświetlane są informacje o wolumenie oraz szczegółowe instrukcje dotyczące połączenia z wolumenem, w tym poszczególne parametry.
+
 ![Zarządzanie wolumenami](images/manage_enterprise04.png){.thumbnail}
 
-W karcie `Informacje ogólne`{.action} wyświetlają się szczegóły wolumenu oraz szczegółowe instrukcje dotyczące połączenia z wolumenem, w tym parametry indywidualne.
+W tej zakładce możesz również zmienić nazwę i opis wolumenu.
+
+### Usuwanie woluminu <a name="delete_volume"></a>
+
+Możesz usunąć wolumen, klikając przycisk`...`{.action} w tabeli w zakładce `Volumes`{.action}, a następnie `Usuń wolumen`{.action}.
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]:
```suggestion
Możesz usunąć wolumen, klikając przycisk `...`{.action} w tabeli w zakładce `Volumes`{.action}, a następnie `Usuń wolumen`{.action}.
```

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md [hunk 13]
```diff
@@ -41,8 +37,10 @@ Możesz wykonać kilka operacji klikając przycisk...` `{.action} w każdym wier
 
 - **Zmień wolumen**: otwiera sekcję "[Informacje ogólne](#modify_volume)" wolumenu.
 - **Utwórz snapshot**: otwiera sekcję "[Kopie](#snapshots) zapasowe", aby wykonać ręczny snapshot wolumenu.
+- **Przywróć ostatni snapshot**: Otwiera sekcję "[Snapshots](#snapshots)" oraz okno umożliwiające przywrócenie woluminu.
 - **Zarządzanie snapshotami**: otwiera sekcję "[Kopie zapasowe](#snapshots)" wolumenu.
 - **Zarządzanie IP Access (ACL)**: otwiera sekcję "[ACL](#access_control)", która umożliwia zarządzanie kontrolą dostępu do wolumenu.
+- **Zmień rozmiar**: Otwiera okno umożliwiające zmianę rozmiaru woluminu.d
```
Comment (Kazarus4) [inline pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]: .d I think there're a systran bug translation

Comment (Kazarus4) [reply pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]: ".d"

**Review by Kazarus4 — COMMENTED** [review]
my review done

**Review by Kazarus4 — COMMENTED** [review]
my review done

File: pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md (hunk 10 above)
Comment (Y0Coss) [reply pages/storage_and_backup/file_storage/enterprise_file_storage/netapp_control_panel/guide.pl-pl.md]: anchor shouldn't be translated too

**Pull request merged**